flow/layers/physical_shape_layer.h

```diff
@@ -37,11 +37,18 @@ class PhysicalShapeLayer : public ContainerLayer {
 
 #if defined(OS_FUCHSIA)
   void UpdateScene(SceneUpdateContext& context) override;
+
+  void UpdateSceneChildren(SceneUpdateContext& context);
```

Comment:
I just realized that `UpdateSceneChildren` is `protected` in other layers but `public` in here. Shall we make them consistent as I don't see why this needs to be public. Also, it might be nice to add some documentation to these function declarations. Although we lack the documentation traditionally (just like tests), we're pushing for better documentations and tests for all new code.

Reply:
You can avoid overriding it entirely if you follow one of my suggestions above

Reply:
Removed the separate method and moved inline, which removed a member variable at the cost of a little extra duplication.